ops-bedrock/docker-compose.yml

```diff
@@ -163,7 +165,7 @@ services:
       context: ../
       dockerfile: ./op-geth-proxy/Dockerfile
     ports:
-      - "9090"
+      - "9090:9090"
```

Comment:
I think this might break the e2e tests, which depends on these services being assigned a random free host port at startup, so that we can run tests in parallel. Although I did set `-parallel 1` in the Makefile and CI config, so maybe this is fine?

Unfortunately I don't know if there's a way to make it configurable whether or not an explicit host port is assigned. The best we could maybe do is make the host port an env var and randomly pick one in the test setup code. But maybe this is not worth doing if we're not going to be running tests in paralle.

Reply:
Thanks for the heads up!